src/main/views/courts/tabs/openingHoursContent.njk

```diff
@@ -77,30 +81,55 @@
     errorMessage: hoursError
   }) }}
 
-  {% if isNew === false %}
-    {{ govukButton({
-      text: "Remove",
-      name: "deleteOpeningHours",
-      type: "button",
-      classes: "govuk-button--warning deleteOpeningTime"
-    }) }}
-  {% else %}
-    {{ govukButton({
-      text: "Clear",
-      name: "clearOpeningTime",
-      type: "button",
-      classes: "govuk-button--secondary clearOpeningTime"
-    }) }}
-  {% endif %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-three-quarters">
+      {% if isNew === false %}
+        {{ govukButton({
+          text: "Remove",
+          name: "deleteOpeningHours",
+          type: "button",
+          classes: "govuk-button--warning deleteOpeningTime"
+        }) }}
+      {% endif %}
+
+      {% if isNew === true %}
+        {{ govukButton({
+          text: "Clear",
+          name: "clearOpeningTime",
+          type: "button",
+          classes: "govuk-button--secondary clearOpeningTime"
+        }) }}
+      {% endif %}
+    </div>
+
+    <div class="govuk-grid-column-one-quarter govuk-button-group">
+      {{ govukButton({
+        html: upArrowHtml,
+        name: "moveUp",
+        type: "button",
+        classes: "govuk-button--secondary move-up",
+        attributes: { "aria-label": "move up" }
+      }) }}
+
+      {{ govukButton({
+        html: downArrowHtml,
+        name: "moveDown",
+        type: "button",
+        classes: "govuk-button--secondary move-down",
+        attributes: { "aria-label": "move down" }
+      }) }}
+    </div>
+  </div>
```

Comment:
I wonder if we should really make this a shared component. It could save on repeating this code across each tab.